<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NAIM" failures="2" tests="15" name="Tutorial Ninja functional Test" time="47.326" errors="0" timestamp="2023-09-26T04:36:41 EDT">
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="draftfail" time="9.841">
    <skipped/>
  </testcase> <!-- draftfail -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC001" time="13.437"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality" name="r1dot1_TS003_TC001" time="14.894"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality" name="r1dot2_TS006_TC001" time="15.936"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC001" time="17.016"/>
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="draftfail" time="7.099">
    <skipped/>
  </testcase> <!-- draftfail -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="draftfail" time="5.59">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:23)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- draftfail -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality" name="r1dot2_TS005_TC001" time="10.113"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC003" time="10.523"/>
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="liginPagefail" time="5.325">
    <skipped/>
  </testcase> <!-- liginPagefail -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="liginPagefail" time="4.739">
    <skipped/>
  </testcase> <!-- liginPagefail -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC004" time="8.5"/>
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="liginPagefail" time="5.107">
    <failure type="java.lang.AssertionError" message="expected [Customer] but found [Account Login]">
      <![CDATA[java.lang.AssertionError: expected [Customer] but found [Account Login]
at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:32)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- liginPagefail -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="readExcelDraft" time="1.009"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC028" time="10.065"/>
</testsuite> <!-- Tutorial Ninja functional Test -->
